src/components/stretchable-link/stretchable-link.tsx

```diff
@@ -0,0 +1,27 @@
+import classNames from 'classnames';
+import styles from './stretchable-link.module.css';
+import { Link } from '@remix-run/react';
+import { ROUTES } from '~/router/config';
+
+interface StretchableLinkProps {
+    title: string;
+    categorySlug: string;
+    className?: string;
+    children?: React.ReactNode;
+}
+
+export const StretchableLink = ({ title, className, categorySlug }: StretchableLinkProps) => {
+    return (
+        <Link className={classNames(styles.root, className)} to={ROUTES.products.to(categorySlug)}>
+            {title}
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 200" className={styles.icon}>
```

Comment:
If I import `svg` from file using `<img>` I cannot set it certain color (`fill="currentColor"` doesn't work). 
I didn't want also to hardcode fill color in `svg`.

Reply:
What's the point of a having a decorative link that can only link to category?

Reply:
Make a component `LabelWithArrow`, receives children, has the arrow and the animation effect.

Then you can do:
```
<CategoryLink slug="ice-cream">
     <LabelWithArrow>Ice Cream</LabelWithArrow>
</CategoryLink>
```